### PR TITLE
Add com.snowplowanalytics.snowplow/ip_lookups/jsonschema/2-0-0

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: schemas/com.snowplowanalytics.snowplow/ip_lookups/jsonschema/2-0-0 [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+				},
+
+				"isp": {
+					"type": "object",
+					"properties": {
+						"database": {
+							"enum": ["GeoIP2-ISP.mmdb"]
+						},
+						"uri": {
+							"type": "string"
```
Comment (chuwy) [inline schemas/com.snowplowanalytics.snowplow/ip_lookups/jsonschema/2-0-0]: Can be `format: uri`.

File: schemas/com.snowplowanalytics.snowplow/ip_lookups/jsonschema/2-0-0 [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+			"properties": {
+
+				"geo": {
+					"type": "object",
+					"properties": {
+						"database": {
+							"enum": ["GeoLite2-City.mmdb", "GeoIP2-City.mmdb"]
+						},
+						"uri": {
+							"type": "string"
```
Comment (chuwy) [inline schemas/com.snowplowanalytics.snowplow/ip_lookups/jsonschema/2-0-0]: Can be `format: uri`.

**Comment by alexanderdean** [comment]
Scheduled.

**Comment by alexanderdean** [comment]
@BenFradet - please put together the complete PR for R89. Ping @chuwy if anything is unclear.

**Comment by BenFradet** [comment]
incorporated into #753

**Pull request closed without merging**